README.md

```diff
@@ -14,9 +14,9 @@ OpenFunction features include:
 - Providing event management functions for trigger functions.
 - Providing additional functions for function version management, ingress management, etc.
 
-## ☸ Custom Resource Definitions
+## ☸ Kubernetes-native architecture
 
-<div align=center><img src=docs/images/openfunction-overview.svg></div>
+![OpenFunction Architecture](docs/images/openfunction-0.5-architecture.png)
 
```

Comment:
svg format is more clear than png.
and the order of the events part's order is not adjusted